Zip/Zip.psd1

```diff
@@ -111,8 +111,7 @@
 
             # ReleaseNotes of this module
             ReleaseNotes = '
-* Created `New-ZipArchive` function for creating new archives.
-* Created `Add-ZipArchiveEntry` funcgtion for adding files to an existing ZIP archive.
+* Optimized `Add-ZipArchiveEntry` to compress files more than an order of magnitude faster. In our testing, compressing a 190MB folder of 538 files/assemblies went from 2 minutes to 14 seconds.

```

Comment:
> went from 2 minutes to 14 

😮 🚀 🥇

Reply:
Who knew buffered input could make that much of a difference?